manifests/init.pp

```diff
@@ -117,5 +70,4 @@
   class { '::ntp::config': } ~>
   class { '::ntp::service': } ->
   anchor { 'ntp::end': }
```

Comment:
It's crappy for sure, since 4.8 you'll be able to do:

```
contain("foo") -> contain("bar") ~> contain("baz")
```

which will make things a lot better.